docs/project/dogfooding.md

```diff
@@ -31,9 +29,9 @@ To use nightly builds of the entire runtime, follow the steps given in the rest
 
 ## Install prerequisites
 
-1. Acquire the latest nightly .NET SDK by downloading the zip or tarball listed in https://github.com/dotnet/core-sdk#installers-and-binaries (for example, https://dotnetcli.blob.core.windows.net/dotnet/Sdk/master/dotnet-sdk-latest-win-x64.zip ) into a new folder, for instance `C:\dotnet`.
+1. Acquire the latest nightly .NET SDK by downloading and extracting a zip/tarball or using an installer from the [installers and binaries table in dotnet/installer](https://github.com/dotnet/installer#installers-and-binaries) (for example, https://aka.ms/dotnet/net6/dev/Sdk/dotnet-sdk-win-x64.zip).
 
-2. By default, the dotnet CLI will use the globally installed SDK if it matches the major/minor version you request and has a higher revision. To force it to use the locally installed SDK, you must set an environment variable `DOTNET_MULTILEVEL_LOOKUP=0` in your shell. You can use `dotnet --info` to verify what version of the Shared Framework it is using.
+2. By default, the dotnet CLI will use the globally installed SDK if it matches the major/minor version you request and has a higher revision. To force it to use a locally installed SDK, you must set an environment variable `DOTNET_MULTILEVEL_LOOKUP=0` in your shell. You can use `dotnet --info` to verify what version of the Shared Framework it is using.
```

Comment:
Are there a typical/recommended set of commands we can provide, for each relevant OS, that would let someone simply copy/paste each command into their shell? i.e. one to download the bits, one to extract them, one to set whatever environment variables need to be set, etc.?  I'm thinking of something like the step-by-step copy/paste commands available for the official bits.

Reply:
This is the only piece of feedback that I haven't really addressed. However, the table here has RPM installers, EXE bundles, and archives. It's also a couple of download tables (one here and one in installer) that are more interactive than the nuget cli experience. If we want to limit it to say archived variants of the runtime/SDK, I'm happy helping out with that and I'd say another PR may be better.

Reply:
Ok

Reply:
I've created this to follow up with this #43455